tests/Microsoft.Identity.Test.Unit/RequestsTests/IntegratedWindowsAuthUsernamePasswordTests.cs

```diff
@@ -288,6 +288,51 @@ public async Task AcquireTokenByIntegratedWindowsAuthTestAsync()
             }
         }
 
+        [TestMethod]
+        [DeploymentItem(@"Resources\TestMex3rdParty.xml")]
+        [DeploymentItem(@"Resources\WsTrustResponse13.xml")]
+        public async Task AcquireTokenByIntegratedWindowsAuth3rdPartyIDPTestAsync()
```

Comment:
This test seems to be almost the same as `AcquireTokenByIntegratedWindowsAuthTestAsync`. Can the XML file path can be passed into as a parameter to `AcquireTokenByIntegratedWindowsAuthTestAsync` as a data row?